docs/modules/vertiq_4006.rst

```diff
@@ -0,0 +1,21 @@
+********************************
+Vertiq 4006 | Speed Module
+********************************
+
+.. sidebar:: Vertiq 4006
+
+    .. image:: ../_static/Vertiq4006.jpg
+        :alt: Vertiq 4006 
+
+The Vertiq 4006 370Kv and 770Kv are built for high performance commercial UAV and robotic applications. Its default firmware is the **Speed Firmware**.
+This extremely compact, lightweight module is designed for roboticists with strict space and 
```

Comment:
Not sure its really meant for roboticists as a speed module. Might just ditch that part.

Reply:
Can I remove the entire sentence as it's not exactly extremely compact either?

Reply:
Sure.